src/Webapi/Dom/Webapi__Dom__InputEvent.res

```diff
@@ -10,5 +10,62 @@ include Webapi__Dom__UiEvent.Impl({
 @new external make: string => t = "InputEvent"
 @new external makeWithOptions: (string, {..}) => t = "InputEvent"
 
-@get external data: t => string = "data"
+/*
+  Input event type list:
+  https://rawgit.com/w3c/input-events/v1/index.html#interface-InputEvent-Attributes
+*/
+type inputType = [
+  | #deleteByCut
+  | #deleteByDrag
+  | #deleteContent
+  | #deleteContentBackward
+  | #deleteContentForward
+  | #deleteEntireSoftLine
+  | #deleteHardLineBackward
+  | #deleteHardLineForward
+  | #deleteSoftLineBackward
+  | #deleteSoftLineForward
+  | #deleteWordBackward
+  | #deleteWordForward
+  | #formatBackColor
+  | #formatBold
+  | #formatFontColor
+  | #formatFontName
+  | #formatIndent
+  | #formatItalic
+  | #formatJustifyCenter
+  | #formatJustifyFull
+  | #formatJustifyLeft
+  | #formatJustifyRight
+  | #formatOutdent
+  | #formatRemove
+  | #formatSetBlockTextDirection
+  | #formatSetInlineTextDirection
+  | #formatStrikeThrough
+  | #formatSubscript
+  | #formatSuperscript
+  | #formatUnderline
+  | #historyRedo
+  | #historyUndo
+  | #insertCompositionText
+  | #insertFromDrop
+  | #insertFromPaste
+  | #insertFromPasteAsQuotation
+  | #insertFromYank
+  | #insertHorizontalRule
+  | #insertLineBreak
+  | #insertLink
+  | #insertOrderedList
+  | #insertParagraph
+  | #insertReplacementText
+  | #insertText
+  | #insertTranspose
+  | #insertUnorderedList
+]
+
+@get @return(nullable) external data: t => option<string> = "data"
+@get external dataTransfer: t => option<Dom.dataTransfer> = "dataTransfer"
```

Comment:
This needs `@return(nullable)` since this `new InputEvent('beforeinput').dataTransfer` returns `null` in the console also the spec mentions it being null for various operations https://www.w3.org/TR/input-events-2/#overview

- [ ] Add nullable to dataTransfer

Reply:
Our internal bindings are wrong then 😂 I just copied them here

Reply:
It probably never caused an issue since you normally check the inputType before getting the dataTransfer good to fix it now then. 😊